torch-use-gpu.ipynb

```diff
@@ -240,26 +308,34 @@
     "        transforms.Normalize((0.1307,), (0.3081,)),\n",
     "        ])\n",
     "\n",
+    "# Set up a training data set\n",
     "trainset = datasets.FashionMNIST('./data', train=True, download=True,\n",
     "                   transform=train_transform)\n",
     "train_loader = torch.utils.data.DataLoader(trainset, batch_size=batch_size,\n",
     "                                          shuffle=False, num_workers=2)\n",
     "\n",
+    "# Set up a test data set\n",
     "testset = datasets.FashionMNIST('./data', train=False,\n",
     "                   transform=test_transform)\n",
     "test_loader = torch.utils.data.DataLoader(testset, batch_size=batch_size,\n",
     "                                         shuffle=False, num_workers=2)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "f77af8df",
+   "metadata": {},
+   "source": [
+    "Review the dictionary and sample content from the data set:"
```

Comment:
It's not clear whether the data scientists would need to write this dictionary for themselves, or whether this is somewhere in the FashionMNIST data set. @cam-garrison do you or Audrey know the expectation here?